### DISCUSSION TITLE
How to see total number of matches?

### DISCUSSION THREAD

**Discussion by eminence** [community/community#20344] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello!

When you do a search on cs.github.com, you'll see something like this in the results:

> Showing 81 - 100 of 100+ files found (in 101 milliseconds)

Is it possible to see the total number of matching files?

**Comment by doguedogue** [comment, marked as answer]
Maybe with the API but

Can I retrieve all search results?
No, not at this time, but we intend to provide this capability in the future through an API.
(source: https://cs.github.com/about/faq)
